src/auth/auth.ts

```diff
@@ -505,7 +523,7 @@ authStore.subscribe(
   withErrorIgnoring(async (state: AuthState, oldState: AuthState) => {
     if (!oldState.termsOfService.permitsSystemUsage && state.termsOfService.permitsSystemUsage) {
       if (window.Appcues) {
-        window.Appcues.identify(state.terraUser.id, {
+        window.Appcues.identify(userStore.get().terraUser.id, {
```

Comment:
Since this is now from a different store, are we sure that userStore has been updated before an update to authStore calls this function? In loadTerraUser, userStore is updated after authStore.

Reply:
That's a good call, I will update to make sure `userStore` gets updated first where possible.